framework/src/Volo.Abp.Core/Volo/Abp/Check.cs

```diff
@@ -162,4 +162,182 @@ public static ICollection<T> NotNullOrEmpty<T>(ICollection<T> value, [InvokerPar
 
         return value;
     }
+
+    public static Int16 NotNegativeOrZero(
```

Comment:
Please rename all to `Positive`

Reply:
I have implemented the change  in this commit: [aa08d0d](https://github.com/abpframework/abp/pull/12576/commits/aa08d0d5ad86e298e433c15c26a82a36f8a173b9)